### PR TITLE
Various small docs clarifications

### PR DESCRIPTION
**Pull request by ryandawsonuk**
Points that I noticed when running through this and which I felt would've made it clearer for me

### CODE REVIEW

File: instructions/2-SetupJenkins.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Enter your Github Personal Access Token.
 
 Select your github account, then cd4ml-scenarios and click "Create Pipeline"
 
-At this step the pipeline will build, you can select the pipeline to see the individual steps execute. At the end the pipeline should be 'green' indicating that all the steps were successful (shown below).
+At this step the pipeline will build, you can select the pipeline to see the individual steps execute. Jenkins will first index branches, [which can take time](https://support.cloudbees.com/hc/en-us/articles/360055870591-Multibranch-Pipeline-Branch-indexing-job-stuck-and-cannot-be-aborted). It will then checkout the code which [can also take a while](https://stackoverflow.com/questions/36017253/jenkins-git-fetch-timeout). It does get there eventually. At the end the pipeline should run and be 'green' indicating that all the steps were successful (shown below). If it times out then trigger a manual run.
```
Comment (ryandawsonuk) [inline instructions/2-SetupJenkins.md:21]: This is basically reporting my experience

File: instructions/housing/3-MachineLearning.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ being changed to something else.
 
     Note in the lasso directory, there is a file called big_alpha.json which
 corresponds to the 'big_alpha' that we typed into that text box. It has 
-an alpha parameter of 50 rather than 1 and so encourages sparseness 
+an alpha parameter of 50 rather than 1 and so [encourages sparseness](https://chrisalbon.com/code/machine_learning/linear_regression/effect_of_alpha_on_lasso_regression/)
```
Comment (ryandawsonuk) [inline instructions/housing/3-MachineLearning.md:38]: extra link offers clarification

File: instructions/housing/3-MachineLearning.md [hunk 3]
```diff
@@ -47,21 +47,24 @@ able to find those parameters and build that particular model.
...
-UI as well but need to make another edit in cd4ml/problems/houses/problem.py. Add another
+UI as well but need to make another edit in cd4ml/problems/houses/problem.py. To do this, add another
 option around line 34 to import your new feature set if is to be selected. Unlike 
-algorithms and params, it won't be discovered automatically without that change.
+algorithms and params, it won't be discovered automatically without that change. This happens to be the way we've 
+implemented this but it's not unusual that feature changes require a code change rather than just a run parameter change.
 
 
-6) Looks at the ml_pipeline_params directory. Similarly, it contains a default.json file. 
+6) Look at the ml_pipeline directory. Similarly, it contains a default.json file.
```
Comment (ryandawsonuk) [inline instructions/housing/3-MachineLearning.md:67]: There is no ml_pipeline_params directory, it means ml_pipeline

File: instructions/housing/3-MachineLearning.md [hunk 4]
```diff
@@ -47,21 +47,24 @@ able to find those parameters and build that particular model.
 
 4) Go back Jenkins and observe that the model succeeded. Note that 
 it was identified as an experiment due to using non-default parameters. 
-And so, the pipeline skipped the production check.
+The pipeline therefore skipped the production 'register model' step and ran the experiment step instead.
 
 ![GreenExperiment](../images/GreenExperiment.png)
 
+This lets us use Jenkins like an [ML training orchestration platform](https://github.com/EthicalML/awesome-production-machine-learning#model-training-orchestration).
+
```
Comment (ryandawsonuk) [inline instructions/housing/3-MachineLearning.md:55]: This note helps to clarify that our use of Jenkins here is just an example/illustration

**Comment by ericnagler** [comment]
Looks great. Thanks for the update @ryandawsonuk !

**Pull request merged**